spec/td/logger/agent/rack_spec.rb

```diff
@@ -1,5 +1,6 @@
 require 'spec_helper'
 require "rack/mock"
+require 'rack/builder'
```

Comment:
Added this line because my local test failed due to uninitialized constant.

```
Failures:

  1) TreasureData::Logger::Agent::Rack::Hook.before
     Failure/Error:
       @app ||= Rack::Builder.new {
         use TreasureData::Logger::Agent::Rack::Hook
         run lambda {|env| [200, {}, ["body"]]}
       }.to_app

     NameError:
       uninitialized constant Rack::Builder
     # ./spec/td/logger/agent/rack_spec.rb:7:in `app'
     # ./spec/td/logger/agent/rack_spec.rb:15:in `block (2 levels) in <top (required)>'
     # ./spec/td/logger/agent/rack_spec.rb:22:in `block (3 levels) in <top (required)>'

  2) TreasureData::Logger::Agent::Rack::Hook.after
     Failure/Error:
       @app ||= Rack::Builder.new {
         use TreasureData::Logger::Agent::Rack::Hook
         run lambda {|env| [200, {}, ["body"]]}
       }.to_app

     NameError:
       uninitialized constant Rack::Builder
     # ./spec/td/logger/agent/rack_spec.rb:7:in `app'
     # ./spec/td/logger/agent/rack_spec.rb:15:in `block (2 levels) in <top (required)>'
     # ./spec/td/logger/agent/rack_spec.rb:33:in `block (3 levels) in <top (required)>'

Finished in 0.30078 seconds (files took 1.03 seconds to load)
147 examples, 2 failures, 5 pending
```